src/server/middleware/addFrameOptionsHeader.js

```diff
@@ -21,8 +21,8 @@ export default function addFrameOptionsHeader(req, res, next) {
   const referer = req.get('Referer');
 
   const frameAncestorDomains = getCSP()['frame-ancestors'];
-
-  const matchedDomain = frameAncestorDomains && frameAncestorDomains.find((domain) => matcher.isMatch(referer, `${domain}/*`)
+  const trimmedReferrer = referer && referer.replace('https://', '');
```

Comment:
because if the referrer is `https://some.domain.com` it will always fail to match against `some.domain.com` . i think it is safer to strip https:// instead of putting a wildcard in front of the csp domain.